doc/source/user_guide/meshing/new_meshing_workflows.rst

```diff
@@ -726,7 +577,10 @@ Use the ``mark_as_updated()`` method to forcefully mark a task as updated.
 Sample use of ``arguments``
 ----------------------------
 This simple example shows how to use the ``arguments`` attributes and explicit
-attribute access methods in a watertight geometry meshing workflow.
+attribute access methods in a watertight geometry meshing workflow. Unlike the
+classic meshing workflow, the arguments in the new workflow are fully populated
+with all parameter values that will be used during command execution, rather
+than only those explicitly set by the user.
```

Comment:
We can avoid use of `will` as follows:

Unlike the classic meshing workflow, the arguments in the new workflow are fully populated with all parameter values used during command execution, rather than only those explicitly set by the user.